receiver/windowsperfcountersreceiver/config.go

```diff
@@ -46,23 +69,53 @@ func (c *Config) Validate() error {
 		errs = multierr.Append(errs, fmt.Errorf("must specify at least one perf counter"))
 	}
 
+	if len(c.MetricMetaData) == 0 {
+		errs = multierr.Append(errs, fmt.Errorf("must specify at least one metric"))
+	}
+
+	for name, metric := range c.MetricMetaData {
+		if metric.Unit == "" {
+			metric.Unit = "1"
+		}
+
+		if (metric.Sum != SumMetric{}) {
+			if metric.Sum.Aggregation != "cumulative" && metric.Sum.Aggregation != "delta" {
```

Comment:
Don't we also need to check temporality?

Reply:
not sure what you mean? this check is checking the temporality?

Reply:
Sorry, I meant non/monotonic, but see that it is a boolean so can't be nil.